mechafil/data.py

```diff
@@ -4,198 +4,108 @@
 import datetime
```

Comment:
The `requests` library is no longer used here. We can delete it

Reply:
done